techniques/aria/aria25.html

```diff
@@ -0,0 +1,194 @@
+				<p>To meet 3.3.2 ("Labels or instructions are provided when content requires user input."), each of the fields can be given a unique name through the use of the <code>title</code> attribute. Where an accessible name already exists (for the first input), the <code>title</code> value becomes part of the accessible <em>description</em>. In the absence of another candidate for accessible name, the <code>title</code> value is used in the calculation for the name (for the second and third inputs).</p>
+				<code>
+&lt;div role="group" aria-labelledby="groupLabel"&gt;<br>
+ &lt;span id="groupLabel"&gt;Work Phone:&lt;/span&gt;<br>
+ &lt;input type="number" aria-labelledby="groupLabel" title="Area Code" min="3" max="3" /&gt;<br>
+ &lt;input type="number" title="Prefix" min="3" max="3" /&gt;<br>
+ &lt;input type="number" title="Line Number" min="4" max="4" /&gt;<br>
+&lt;/div&gt;<br>
+
+				</code>
+				<p>There are several advantages to constructing the inputs this way, some of which go beyond the requirements of 2.5.3 but are relevant when considering the most appropriate techniques:
+					</p><ol>
+						<li>Speech-input users who say "move to work phone" will be moved to the first of the three fields, which is where they would typically wish to begin. If they wish to reposition to the other inputs, it is simple for them to issue a "Press tab" command.</li>
+						<li>Sighted users who wish to have more context can use hover to reveal the <code>title</code> value as a tool tip. This can be beneficial for users with low vision or with some cognitive disabilities.</li>
+						<li>Since the <code>title</code> attribute is used across all three inputs, the behaviour of the inputs is consistent for many users.</li>
+						<li>The computation for accessible name and description ensures that the value of <code>title</code> is always available programmatically, either as the accessible name or description. Thus screen readers will always have access to this information. </li>
+					</ol>
+				<p></p>
+				<p>There are a couple of disadvantages to this construction:
+					</p><ol>
+						<li>Screen reader users will hear "Work Phone:" announced twice – as the group name and as the label of the first field. (This is an unfortunate outcome of the new 2.5.3 requirement and will occur for at least one input in a set of inputs with a group label in all implementations.)</li>
+						<li>The keyboard user cannot see a label for the second and third inputs since most browsers do not provide a mechanism to display <code>title</code> via keyboard. (Unless the telephone inputs are redesigned to provide persistently visible labels, this will remain a challenge.)</li>
+					</ol>
+				<p></p>
+				<p class="working-example"><a href="file:///C:/Users/IBM_ADMIN/Documents/GitHub/wcag/working-examples/label-in-name-general/example3.html" target="_blank">Working example of Work Phone input set</a></p>
```

Comment:
Removing reference to working example, since I know longer have it locally. Easy enough to add later
```suggestion
```